src/main/webapp/app/course/competencies/competencies-popover/competencies-popover.component.ts

```diff
@@ -12,7 +12,7 @@ export class CompetenciesPopoverComponent implements OnInit {
     @Input()
     courseId: number;
     @Input()
-    competencies: Competency[] = [];
+    competencyLinks: CompetencyLectureUnitLink[] = [];
     @Input()
     navigateTo: 'competencyManagement' | 'courseCompetencies' = 'courseCompetencies';
```

Comment:
No, but I would consider this out of scope for this PR. Especially since we are already over 1500 new/changed lined of code and this would add hundreds more